src/languages/es.ts

```diff
@@ -1021,6 +1021,7 @@ const translations = {
         bookingPendingDescription: 'Esta reserva está pendiente porque aún no se ha pagado.',
         bookingArchived: 'Esta reserva está archivada',
         bookingArchivedDescription: 'Esta reserva está archivada porque la fecha del viaje ha pasado. Agregue un gasto por el monto final si es necesario.',
+        paymentComplete: 'Pago completo',
```

Comment:
Did a Spanish speaker double check this translation? 
<img width="1317" alt="image" src="https://github.com/user-attachments/assets/702b639f-51e8-4829-bfa3-47f91701fe27">

Maybe there's something better?

Reply:
used [this](https://expensify.slack.com/archives/C01GTK53T8Q/p1727996023007789) 
and it gives me a different response

`Pago completado.`

Reply:
![Screenshot 2024-10-10 at 5 21 15 in the afternoon](https://github.com/user-attachments/assets/b95f4d51-6e31-4eea-b72e-437e245b2298)

Reply:
I trust native Spanish speakers more than ChatGPT and Google Translate 🙃

Reply:
So I'm happy that Alberto approved it (he is Spanish)